openid4vc-high-assurance-interoperability-profile-sd-jwt-vc-1_0.md

```diff
@@ -198,7 +206,7 @@ This is an example of a Wallet Instance Attestation:
 
 * The Credential Issuer MUST publish a mapping of every Credential Type it supports to a scope value.
 
-# OpenID for Verifiable Presentations
+# OpenID for Verifiable Presentations without using W3C Digital Credentials API
```

Comment:
I could live with 'over HTTPS'.

I definitely think we should clarify it though, as it otherwise looks very strange having sections for both:

> OpenID for Verifiable Presentations

and:

> OpenID for Verifiable Presentations over W3C Digital Credentials API

I think people would get confused as to whether the clauses in the VP section also apply when using the VP over W3C API section.